index.html

```diff
@@ -16,16 +19,139 @@
 
 <body>
 
-    <div class="container">
-      <div class="row">
-          <div class="col-sm-12">
-              <h1>Start Here!</h1>
+  <div class="container">
+
+    <div class="row">
+      <div class="col-sm-12">
+        <div class="jumbotron">
+          <div class="row">
+            <div class="col-sm-12">
+
+              <nav class="nav">
+
+                <a class="nav-link active" href="#">
+                  <i class="fas fa-user"></i> About Me</a>
+                <a class="nav-link" href="#">
+                  <i class="far fa-keyboard"></i> Experience</a>
+                <a class="nav-link" href="#">
+                  <i class="far fa-folder-open"></i> Portfolio</a>
+                <a class="nav-link" href="#">
+                  <i class="fas fa-phone-square"></i> Contact</a>
+              </nav>
+            </div>
+            <!-- col-sm-12 -->
+          </div>
+          <!-- row -->
+          <h1 class="display-4">Justin M. Rigler
+            <span class="badge badge-secondary">Updated</span>
+          </h1>
+          <img src="img/profile-img.jpg" class="img-fluid" alt="me not at denny's">
+          <p class="lead">I am a content creation specialist who focuses on full-scale web development and deployment. With your vision,
+            and my expertise, we will change the world.</p>
+          <hr class="my-4">
+          <p>Feel free to use the navigation bar at the top of this page to navigate through the sections we have conveniently
+            outlined for your perusal.</p>
+        </div>
+        <!-- jumbotron -->
+      </div>
+      <!-- col-sm-12 -->
+    </div>
+    <!-- row -->
+
+    <div class="top-content">
+    <div class="row">
+      <div class="col-sm-8">
+        <h2>
+          <i class="fas fa-user"></i> Developer</h2>
+        <p>Being a developer is about the thrill of the build, the passion that's needed to commit to project after project.
```

Comment:
Nice work on the colors and even the fonts! Very clean and simple. One thing I would recommend here is that your <p> are touching left side of the container. You can just simply add a margin left in your css to fix that. This also goes for your <div class="col-sm-4"> table!